test/e2e/playwright.config.ts

```diff
@@ -0,0 +1,37 @@
+import type { PlaywrightTestConfig } from '@playwright/test'
+
+const config: PlaywrightTestConfig = {
+  testDir: '.',
+  webServer: {
+    /**
+     * Note: You can speed up local testing by switching `prod` out for `start`
+     * to skip the build, but be aware that then talkback and the Nuxt server
+     * will be racing to warm up and the tests could start before talkback is
+     * ready to start serving API responses. Playwright only lets you specify
+     * one port to wait for a ready response from so there's no way around it.
+     * Normally the build introduces more than enough time for talkback to
+     * be ready by the time the Nuxt server is actually started.
+     *
+     * This doesn't mean _don't_ use `start`, it just means that once you've
+     * debugged everything else, make sure to inspect the trace output for any
+     * ConsoleMessage strings that refer to incorrect responses. Once you've
+     * verified that there is indeed a tape saved for the response in question,
+     * switch this back to `prod` and see if your tests pass.
+     */
+    command: './node_modules/.bin/npm-run-all -p -r talkback prod',
+    timeout: 120000,
+    port: 8443,
+    reuseExistingServer: !process.env.CI || process.env.PWDEBUG === '1',
+    env: {
+      API_URL: 'http://localhost:3000',
```

Comment:
Should we change this port value to `49152` as in #1045, or does it not matter when inside Docker?

Reply:
Oh, I had a comment about changing this once you PR is merged.

It needs to match whatever talkback is using so once your PR is merged, if it goes before this one, we need to update it to the new port number you chose.